<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogCutter</class>
 <widget class="QWidget" name="LogCutter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Cutter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="inputList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="sourceAddBtn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sourceRemoveBtn">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sourceClearBtn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputFile"/>
     </item>
     <item>
      <widget class="QToolButton" name="outputSelectBtn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cutNonGameChk">
     <property name="text">
      <string>Remove pre/post game and half times</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cutHaltChk">
     <property name="text">
      <string>Remove halt and timeouts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cutStop">
     <property name="text">
      <string>Remove stop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cutBallplacement">
     <property name="text">
      <string>Remove ball placement</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cutSimulated">
     <property name="text">
      <string>Remove simulated game parts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cutGit">
     <property name="text">
      <string>Remove all git infos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="processBtn">
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="progressLbl">
     <property name="text">
      <string>Idle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
